src/main/java/com/github/nylle/javafixture/Reflector.java

```diff
@@ -64,6 +69,26 @@ private Stream<Field> getDeclaredFields(Class<?> type) {
                         .orElse(Stream.of()));
     }
 
+    public Annotation[] getFieldAnnotations(Field field) {
```

Comment:
Please add tests for this method, at least one for the success case and one for the exception case.

Reply:
Or alternatively, I can do that, of course.

Reply:
Ah, it's alright, I'll do it real quick...